### PR TITLE
refactor(): one ui port and use case per feature

### PR DESCRIPTION
**Pull request by svoren258**

### CODE REVIEW

File: src/app/libs/spaces/infrastructure/tokens.ts [hunk 1]
```diff
@@ -17,23 +10,8 @@ export const INFRASTRUCTURE_PROVIDERS = [
...
-  },
-  {
-    provide: LoadSpacesUiPort,
-    useExisting: LoadSpacesUseCase,
-    deps: [SpacesDataPort]
-  },
-  {
-    provide: ShowSettingsUiPort,
-    useExisting: ShowSettingsUseCase,
-    deps: [SettingsApi]
```
Comment (svoren258) [inline src/app/libs/spaces/infrastructure/tokens.ts:37]: Not sure about the dependency on the `SettingsApi`. 🤔

File: src/app/libs/sidebar/infrastructure/tokens.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { SidebarDataPort, ShowSettingsUiPort, ShowSettingsUseCase } from '../use-cases';
+import { SidebarDataPort, SidebarUiPort, SidebarUseCase } from '../use-cases';
 import { SidebarStoreService } from './sidebar-store.service';
 
 export const INFRASTRUCTURE_PROVIDERS = [
```
Comment (maximal01) [inline src/app/libs/sidebar/infrastructure/tokens.ts:4]: I don't know how this is still working without providing `deps` array with `SettingsApi`. @hoebbelsB maybe you have an idea?

**Comment by maximal01** [comment]
Maybe we should rename `use-cases/ports/infrastructure` to `use-cases/ports/data`?

File: src/app/libs/sidebar/infrastructure/tokens.ts (hunk 2 above)
Comment (svoren258) [reply src/app/libs/sidebar/infrastructure/tokens.ts:4]: Yes 😅 that's what I was curious about.

**Comment by svoren258** [comment]
> Maybe we should rename `use-cases/ports/infrastructure` to `use-cases/ports/data`?

Sounds reasonable. Or just rename the file (for example, `spaces-infrastructure.port.ts`) as long as we are using the keyword `infrastructure` also for the providers in `tokens.ts`. 🤔

**Review by hoebbelsB — APPROVED** [review]

**Pull request merged**